src/main/resources/catalog.json

```diff
@@ -0,0 +1,181 @@
+    "id" : 1,
+    "name" : "login",
+    "displayName" : "Login",
+    "description" : "User Login",
+    "aliases" : [ ],
+    "attributes" : [ {
+      "name" : "timestamp",
+      "required" : true
+    }, {
+      "name" : "userId",
+      "required" : true
+    }, {
+      "name" : "requestId",
+      "required" : false
+    } ]
+  }, {
+    "id" : 2,
+    "name" : "updateKey",
+    "displayName" : "Update Key",
+    "description" : "User updates API key(s)",
+    "aliases" : [ ],
+    "attributes" : [ {
+      "name" : "requestId",
+      "required" : false
+    }, {
+      "name" : "requestUri",
+      "required" : false
+    }, {
+      "name" : "userId",
+      "required" : true
+    }, {
+      "name" : "timestamp",
+      "required" : true
+    } ]
+  }, {
+    "id" : 3,
+    "name" : "updatePassword",
+    "displayName" : "Update Password",
+    "description" : "User updates password",
+    "aliases" : [ ],
+    "attributes" : [ {
+      "name" : "requestUri",
+      "required" : false
+    }, {
+      "name" : "userId",
+      "required" : true
+    }, {
+      "name" : "requestId",
+      "required" : false
+    }, {
+      "name" : "timestamp",
+      "required" : true
+    } ]
+  }, {
+    "id" : 4,
+    "name" : "logout",
+    "displayName" : "Logout",
+    "description" : "User Logout",
+    "aliases" : [ ],
+    "attributes" : [ {
+      "name" : "timestamp",
+      "required" : true
+    }, {
+      "name" : "userId",
+      "required" : true
+    }, {
+      "name" : "requestId",
+      "required" : false
+    } ]
+  }, {
+    "id" : 5,
+    "name" : "createUser",
+    "displayName" : "Create User",
+    "description" : "Create User",
+    "aliases" : [ ],
+    "attributes" : [ {
+      "name" : "userId",
+      "required" : true
+    }, {
+      "name" : "requestId",
+      "required" : false
+    }, {
+      "name" : "timestamp",
+      "required" : true
+    }, {
+      "name" : "requestUri",
+      "required" : false
+    } ]
+  } ],
+  "attributes" : [ {
+    "id" : 3,
+    "name" : "userId",
+    "displayName" : "UserId",
+    "description" : "Id of the User",
+    "dataType" : "INT",
+    "indexed" : true,
+    "sortable" : true,
+    "required" : true,
+    "requestContext" : true,
+    "examples" : null,
+    "aliases" : [ ],
+    "constraints" : [ ],
+    "catalogId" : "DEFAULT"
+  }, {
+    "id" : 11,
+    "name" : "requestId",
+    "displayName" : "RequestId",
+    "description" : "Used as a correlation id for logs",
+    "dataType" : "STRING",
+    "indexed" : true,
+    "sortable" : true,
+    "required" : true,
+    "requestContext" : true,
+    "examples" : null,
+    "aliases" : [ ],
+    "constraints" : [ ],
+    "catalogId" : "DEFAULT"
+  }, {
+    "id" : 12,
+    "name" : "requestUri",
+    "displayName" : "Request URI",
+    "description" : "Contains the original request URI used for action",
+    "dataType" : "STRING",
+    "indexed" : false,
+    "sortable" : true,
+    "required" : false,
```

Comment:
At first, I was thinking this should be required, but then I thought that there might not be a logical URI for CLI commands. What do you think about adding a similar attribute named something like `commandExecuted`? We might also end up creating audit log events for CLI commands explicitly.

Reply:
Yes. Right now, I think there are no events listed where this attribute will be used. Am I understanding it correctly?

Reply:
Oh that's a good point. There aren't really any auth-related commands in the CLI; instead, commands usually take an `-auth` parameter with your API token or password specified.

Reply:
These are common attributes that we'll be adding data for on every HTTP request.